packages/fiori/src/UploadCollectionItem.js

```diff
@@ -333,6 +384,10 @@ class UploadCollectionItem extends ListItem {
 		this.fireEvent("terminate");
 	}
 
+	get list() {
+		return this.assignedSlot.parentElement;
```

Comment:
Attach events from the list and fire them from the item if you want the list to take any action